src/components/TheHome.vue

```diff
@@ -1,5 +1,7 @@
 <template>
-  <div></div>
+  <div><LandingHero></LandingHero></div>
```

Comment:
I'm not sure about the routing, should we do more research or ask the teachers about this? Thoughts?